smart-contracts/contracts/mixins/MixinLockCore.sol

```diff
@@ -51,6 +51,9 @@ contract MixinLockCore is
   // The account which will receive funds on withdrawal
   address public beneficiary;
 
+  // The denominator component for values specified in basis points.
```

Comment:
I'm unclear why a denominator is needed if we're switching to basis points to get away from fractions?

Reply:
we want to support values < 1.  so it's still fractions under the hood, I'm just hard-coding the denominator to simplify the api

Reply:
Ahh, I think I get it. This value seems to get used as the denominator in a few calculations below, so it's just a naming thing.